src/services/partnerServices.js

```diff
@@ -65,6 +66,13 @@ class PartnerService {
       data
     );
   }
+
+  updatePendingPartner(data) {
```

Comment:
no need to create new service it will duplicate, please use existing 'update(data)' service